.changeset/great-crews-own.md

```diff
@@ -0,0 +1,19 @@
+---
+'@backstage/cli': patch
+---
+
+Add support for overriding the `testFilePatterns` used when generating eslint config.
+
+If `testFilePatterns` is supplied, the patterns will be used instead of the defaults when applying test-specific linting configuration. Note that the test running configuration in Backstage is opinionated about which files contain _tests_ - with that in mind, this option is mostly useful for supplying patterns which match test support files (utils, mocks, etc).
```

Comment:
> this option is mostly useful for supplying patterns which match test support files (utils, mocks, etc)

To be honest this is something that's missing from the current config. What patterns would use use? I'd be happy to try to choose something to include in config by default.

Reply:
Yeah - maybe we should just add `**/__tests__/**` and `**/__mocks__/**` to the list and leave customization out of it, WDYT?

Reply:
Hmm, yeah I think we could do something like that. I do believe we need a bit more though right? Don't think we'll be allowed to export from those files with the current setup?